includes/wf_crm_webform_base.inc

```diff
@@ -527,10 +529,15 @@ abstract class wf_crm_webform_base {
       );
     }
     $file = wf_crm_apivalues('file', 'get', $val);
+    $entity_id = '';
+    if ($entity && $n && ( strpos($fieldName, 'custom_') === 0 || strpos($fieldName, 'file_') === 0)) {
+      $entity_id = $this->ent[$entity][$n]['id'];
```

Comment:
do we really need this functionality for entering a new contact? this is only for showing download link.